src/plotApplications/helpers.js

```diff
@@ -19,5 +23,59 @@ export const getContentApplicationListItem = (plotApplication: Object): Object =
   return {
     id: plotApplication.id,
     plot_search: plotApplication.plot_search,
+    applicant: plotApplication.applicant,
+    plot_search_type: plotApplication.plot_search_type,
+    plot_search_subtype: plotApplication.plot_search_subtype,
+    target_identifier: plotApplication.targets.map(target => target.identifier),
+    target_address: plotApplication.targets.map(target => target.address.address),
+    target_reserved: plotApplication.targets.map(target => target.reserved),
+  };
+};
+
+/**
+ * Get application target features for geojson data
+ * @param {Object[]} targets
+ * @returns {Object[]}
+ */
+export const getApplicationTargetFeatures = (targets: Array<Object>): Array<LeafletFeature> => {
+  return targets.map((target) => {
+    const coordinates = [];
+    const areas = get(target, 'targets', []);
+
+    areas.forEach((area) => {
+      const coords = get(area, 'geometry.coordinates', []);
+
+      if(coords.length) {
+        coordinates.push(coords[0]);
+      }
+    });
+
+    return {
+      type: 'Feature',
+      geometry: {
+        coordinates: coordinates,
+        type: 'MultiPolygon',
+      },
+      properties: {
+        id: target.id,
+        feature_type: 'plotApplication',
+        identifier: target.targets[0].identifier,
```

Comment:
This doesn't work. If the application is applying for more than one target, the map will visually show all of its areas but then link to listing based on the first target whenever any of them is clicked. If the topmost shape drawn has a different first target, the link will be entirely unrelated.

An array of features, one for each target (i.e. mapping "target.targets", actually "application.targets") should be returned here instead. (Whether the same target across different applications should get multiple overlaid features is another matter, I got 9 on top of each other which results in a rather opaque shape.)